### PR TITLE
Updated references page for bib.ietf.org and better advice

### PR DESCRIPTION
**Pull request by JayDaley**
@ajeanmahoney - For someone in the RPC to check and approve or return to sender please, particularly the new advice on line 30.

### CODE REVIEW

File: references-in-rfcxml.md [hunk 1]
```diff
@@ -244,7 +244,6 @@ and <a href="https://www.rfc-editor.org/rfc/rfc3550#section-6">6</a> in
 ```
 
 # Supporting tools
-Authors who often cite academic publications may find the following tool helpful, since most databases include bibliographic entries in BibTex format:
 
 ## bibtex2rfc
-[bibtex2rfc](https://github.com/yaronf/bibtex2rfc) converts BibTeX references into valid RFCXML references.
+Authors who often cite academic publications may find the [bibtex2rfc](https://github.com/yaronf/bibtex2rfc) tool, which converts BibTeX references into valid RFCXML references, helpful, since most databases include bibliographic entries in BibTex format.
```
Comment (ajeanmahoney) [inline references-in-rfcxml.md:249]: A comma is not needed before "since".

File: references-in-rfcxml.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ dateCreated: 2021-11-04T23:45:18.949Z
 # Introduction
 RFCXML uses references that are encoded in a format called BibXML, as defined in RFC 7991. To simplify the process of inserting references, the IETF maintains a set of online citation libraries as follows:
 * For RFCs the official site is [rfc-editor.org](https://rfc-editor.org/). The information page for each RFC has a link to the BibXML file.
-* For RFCs, Internet-Drafts, and documents produced by the W3C, 3GPP, IANA and NIST the current site is [xml2rfc.tools.ietf.org]().  Many of these sets of references can be downloaded as a set for offline access.
+* For RFCs, Internet-Drafts, and documents produced by the W3C, 3GPP, IANA and NIST, the new site is [bib.ietf.org](https://bib.ietf.org) (at the time of writing, this site has only just gone live and some usability issues are still to be resolved). Many of these sets of references can be downloaded as a set for offline access.
```
Comment (ajeanmahoney) [inline references-in-rfcxml.md:14]: A big usability issue is that the site doesn't actually provide the URL that an author needs to create an xi:include, but the link to bib.ietf.org does need to be updated on this page. An issue needs to be opened for this page to track these usability issues and then remove this line when we think bib.ietf.org is really ready to go.

**Review by ajeanmahoney — CHANGES_REQUESTED** [review]
Just a small nit that needs to be fixed and a TODO identified.

File: references-in-rfcxml.md (hunk 2 above)
Comment (JayDaley) [reply references-in-rfcxml.md:14]: > A big usability issue is that the site doesn't actually provide the URL that an author needs to create an xi:include, 

Yes indeed - multiple eyeroll emojis.  I will add an issue.

**Pull request merged**